eng/Subsets.props

```diff
@@ -39,6 +39,9 @@
     <RuntimeFlavor Condition="'$(TargetsMobile)' == 'true'">Mono</RuntimeFlavor>
     <RuntimeFlavor Condition="'$(RuntimeFlavor)' == '' and ($(_subset.Contains('+mono+')) or $(_subset.Contains('+mono.runtime+'))) and (!$(_subset.Contains('+clr+')) and !$(_subset.Contains('+clr.runtime+')))">Mono</RuntimeFlavor>
     <RuntimeFlavor Condition="'$(RuntimeFlavor)' == ''">CoreCLR</RuntimeFlavor>
+
+    <!-- this property is used by the SDK to pull in mono-based runtime packs -->
+    <UseMonoRuntime Condition="'$(UseMonoRuntime)' == '' and '$(RuntimeFlavor)' == 'Mono'">true</UseMonoRuntime>
```

Comment:
can you please move that property out of Subsets.props into Directory.Build.props to the feature switches? https://github.com/dotnet/runtime/blob/b52d276e3c18687a120747e2edfd3ae50679683f/Directory.Build.props#L205

Reply:
Done. I couldn't put it in Feature switches since it needs the Subsets.props import which is after that one.